Aptoide/aptoide-client-v8
# Feature/asv 423 set styles in app

**What does this PR do?**

This PR aims to uniformize styles and themes in all the app except for stores. Changed the theme files to make them usable and consistent across and applied them to every view possible; 
Didn't at this stage however clear the unused parts of themes;

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] themes.xml
- [ ] styles.xml

**How should this be manually tested?**

Check everything in the app and verify that buttons and toolbars are showing as expected;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-415](<https://aptoide.atlassian.net/browse/ASV-415>)

**Code Review Checklist**
